carla-carma-integration/carla-carma-vehicle/package.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>carla-carma-vehicle</name>
+  <version>0.0.0</version>
+  <description>The carla-carma-vehicle package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="carma@todo.todo">carma</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
```

Comment:
Apache2 is the license we use for all of our own code, is this all new code in this package or is it based off of existing material?